tests/integration_tests/test_grid_transmissibility_snapshots.py

```diff
@@ -0,0 +1,33 @@
+import os
+from inspect import getsourcefile
+from numpy import loadtxt, savetxt
+
+import resqpy.model as rq
+
+
+def write_snapshot(data, title):
+    # Method to write out a new snapshot of data if one is needed
+    reshaped_array = data.reshape(data.shape[0], -1)
+    savetxt(f'C:\\Test\\{title}.txt', reshaped_array)
+
+
+def check_load_snapshot(data, filename):
+    # Compare the actual data against the stored expected array
+    loaded_array = loadtxt(filename)
+    expected_array = loaded_array.reshape(loaded_array.shape[0], loaded_array.shape[1] // data.shape[2], data.shape[2])
+    assert (data == expected_array).all()
```

Comment:
I think better to use numpy arrays and check with assert_array_almost_equal() seeing as the snapshot data has been stored in ascii.